Sources/XcodeGraph/Models/Target.swift

```diff
@@ -227,22 +227,26 @@ public struct Target: Equatable, Hashable, Comparable, Codable, Sendable {
              .macro,
              .dynamicLibrary,
              .staticLibrary,
-             .staticFramework,
              .xpc:
             return false
+
+        case .staticFramework:
+            return containsResources
         }
     }
 
+    /// Returns true if the target contains resources or CoreData models
+    public var containsResources: Bool {
+        !resources.resources.isEmpty || !coreDataModels.isEmpty
+    }
```

Comment:
I'd keep this inside `TuistCore` as an extension. We're trying to keep the API surface of `XcodeGraph` smaller to make it easier not to make breaking changes here and to limit the need for cross-repo PRs.